packages/lesswrong/components/review/ReviewVotingWidget.tsx

```diff
@@ -24,7 +21,7 @@ const styles = (theme) => ({
   }
 })
 
-const ReviewVotingWidget = ({classes, post, setNewVote}: {classes:ClassesType, post: PostsListBase, title?: React.ReactNode, setNewVote?: (newVote:number)=>void}) => {
+const ReviewVotingWidget = ({classes, post, setNewVote, showTitle=true}: {classes:ClassesType, post: PostsMinimumInfo, showTitle?: boolean, setNewVote?: (newVote:number)=>void}) => {
```

Comment:
currentUserReviewVote was moved to PostsMinimumInfo

"title" was a vestigial prop no longer used, but I turned it into a boolean that toggles whether to show the title text.